kuma/wiki/models.py

```diff
@@ -811,6 +809,9 @@ def revert(self, revision, user, comment=None):
         revision.created = datetime.now()
         revision.creator = user
         revision.save()
+        if revision.document.original == self:
+            revision.based_on = revision
+        revision.save()
```

Comment:
This will trigger the `post_save` signal callbacks, are we should we should be doing that here yet?